main.go

```diff
@@ -20,6 +20,8 @@ var (
 	LastInfo steam.InfoResponse
 )
 
+var first bool
```

Comment:
This does not describe what its for. Renaming to `initializedPresence` would suit better.
Also, since the default value of a bool is false, and we invert the meaning with this name, the initialization to true in `main` would no longer be required.